### PR TITLE
Change of text if party gems still a thing?

### PR DESCRIPTION
**Pull request by eiron**
This was a simple change I drafted ages ago, updated to match current text, which may be entirely wrong. Not sure if party gems are still a thing. If not, then this is unnecessary. Again, apologies if none of this makes sense. Still horrendously new and woefully under-qualified. Probably should not even be poking about here. Yay!

### CODE REVIEW

**Comment by crookedneighbor** [comment]
@Eiron Currently, parties don't have shared gems. It's an interesting idea, but the only benefit I can think of to add party gems would be to pool gems from various members to buy a quest scroll for the party.

I imagine it this way:
1) Member of party clicks on quest scroll, modal pops up asking if the user wants to use own gems, or the party's
2) If party is chosen, an alert is sent to the rest of the members asking to confirm the purchase (like starting a quest).
3) If a certain number of members approve the purchase, maybe a super majority, the scroll is purchased. The leader of the party can choose to just purchase it immediately, if desired.

However, I believe in the near future there's going to be a way to gift items, including gems, so that may be a mute point.

File: views/options/social/group.jade [hunk 1]
```diff
@@ -1,9 +1,17 @@
-a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter', popover-title=env.t('guildBankPop1'), popover=env.t('guildBankPop2'), popover-placement='left', popover-html='true')
```
Comment (crookedneighbor) [inline views/options/social/group.jade:1]: **FYI:** The `env.t('string')` stuff you've removed is important, that's how the strings get translated into different languages.

Comment (lemoness) [inline views/options/social/group.jade:1]: For what it's worth, gifting gems is coming very, very soon.

**Comment by crookedneighbor** [comment]
@Eiron I'm going to close this PR without merging, but please keep working on the project and submit more PRs.

**Pull request closed without merging**